content/logs/graph.md

```diff
@@ -72,12 +72,12 @@ Select or click on a section of the graph to either zoom in the graph or see the
 
 {{< img src="logs/graph/export_button.png" alt="view logs button" responsive="true" popup="true" style="width:40%;">}}
 
-Export your Log graph: 
+Export your Log Graph: 
 
 * To create a new [log monitor][4]:  
-    The query of your log graph is used to create the log monitor query
+    The query of your Log Graph is used to create the log monitor query.
 * To an existing [Timeboard][5]:  
-    This functionality is not released yet, stay tuned ;)
+    This functionality is not released yet–stay  tuned ;)
```

Comment:
I don't think we do winky-faces in our documentation. Suggest removing it and terminating the phrase with a period or, if you're feeling saucy, an exclamation point.